src/cli.ts

```diff
@@ -175,6 +175,10 @@ async function prepareSuites(inputs: string[]) {
    * Validate and handle configs
    */
   const config = readConfig(environment, options.config);
+  /**
+   * Favor playwright options passed via cli to inline playwright options
+   */
+  config.playwrightOptions = merge(config.playwrightOptions, options.playwrightOptions || {});
```

Comment:
would move this here to avoid changing the config object shape. 

https://github.com/elastic/synthetics/blob/fd61e4b5bda46a0746a760a99ccde9e72f685399/src/cli.ts#L187